man/bemenu.1

```diff
@@ -0,0 +1,194 @@
+.TH bemenu 1 2019-08-07
+
+.SH NAME
+bemenu \- Dynamic menu inspired by
+.BR dmenu (1)
+
+.SH SYNOPSIS
+.B bemenu
+.RB [ -hviw ]
+.RB [ -l
+.IR lines ]
+.RB [ -p
+.IR prompt ]
+.RB [ -P
+.IR prefix ]
+.RB [ -I
+.IR index ]
+.RB [ --scrollbar
+.IR when ]
+.RB [ --ifne ]
+.RI [ backend-options ]
+
+.B bemenu-run ...
+
+.SH DESCRIPTION
+.B bemenu
+is a dynamic menu for
+.BR tty (4)
+(using
+.BR ncurses (3)),
+X11 and/or Wayland, inspired by
+.RB dmenu (1).
+It reads a list of newline-separated items from standard input.
+When the user selects an item and presses Return,
+their choice is printed to standard output and bemenu terminates.
+Entering text will narrow the items to those matching the tokens in the input.
+
+.B bemenu-run
+is a special-case invocation of
+.BR bemenu ,
+where the input is a list of executables in the $PATH directories,
+and the selection is executed by the $SHELL.
```

Comment:
the bemenu-run doesn't actually use $SHELL, instead it forks and execs.

Reply:
Corrected. I took too much inspiration from the `dmenu` man page without fact checking it :sweat_smile: